publishing-kube-rules.yaml

```diff
@@ -7,6 +7,145 @@ recursive-delete-patterns:
 - BUILD.bazel
 - "*/BUILD.bazel"
 rules:
+# origin-4.5-kubernetes-1.18.2
```

Comment:
Is it desirable to add new rules vs updating the rules for 1.18.0-rc.1?

Reply:
We remove the old eventually.

Reply:
... when origin rebase merged.